src/WebJobs.Extensions.CosmosDB/Trigger/CosmosDBTriggerAttributeBindingProvider.cs

```diff
@@ -130,6 +130,8 @@ public async Task<ITriggerBinding> TryCreateAsync(TriggerBindingProviderContext
                     leaseCollectionLocation.ConnectionPolicy.PreferredLocations.Add(location);
                 }
 
+                leaseCollectionLocation.ConnectionPolicy.UseMultipleWriteLocations = attribute.UseMultipleWriteLocations;
```

Comment:
Why do you have to set this here and with the client? What if one was true and one was false? 

I'm mostly just curious :-) -- since it doesn't seem like that could happen here...

Reply:
Honestly, just in case. The CFP Library has both settings available and in case there is some internal logic checking one and not the other, I rather set it on both as it doesn't really hurt.